src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -221,6 +229,42 @@ const UploadForm = () => {
             )
           }
 
+          {/* Document type */}
+          <Controller
+            name="version"
+            control={control}
+            // eslint-disable-next-line @typescript-eslint/no-shadow
+            render={({ field, fieldState: { error } }) => (
```

Comment:
what's being shadowed? could we add an alias here to avoid eslint-disable?

Reply:
My guess is `error`?

Reply:
I just copied the components that were already there, including this line. I removed it and it seemed to pass the linter for me, so I removed them all: [e423746](https://github.com/CMS-Enterprise/easi-app/pull/2748/commits/e423746421692fcd612abbe67965fa0ebff61bc5)